### PR TITLE
spec: Fix invalid '%perl' introduced with 3aadc34c; fix missing Test::Compile for os-autoinst

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: docker/travis_test/Dockerfile [hunk 1]
```diff
@@ -110,6 +110,7 @@ RUN zypper in -y -C \
        'perl(Selenium::Remote::Driver)' \
        'perl(Socket::MsgHdr)' \
        'perl(Sort::Versions)' \
+       'perl(Test::Compile)' \
```
Comment (Martchus) [inline docker/travis_test/Dockerfile]: We don't need `Test::Compile` anymore.

Comment (okurz) [reply docker/travis_test/Dockerfile]: Read the commit message, we do :) See https://travis-ci.org/os-autoinst/openQA/jobs/558842956#L611 and look two lines above. This is os-autoinst!

**Comment by Martchus** [comment]
Ok, makes sense.

File: docker/travis_test/Dockerfile (hunk 1 above)
Comment (kalikiana) [reply docker/travis_test/Dockerfile]: If I may make a suggestion: this should go in the description of the PR. Commit messages are several layers deeper and not very visible.

Comment (okurz) [reply docker/travis_test/Dockerfile]: sure. Done retroactively. Normally for a single-commit PR I use https://github.com/okurz/scripts/blob/master/git-pr-last to put the commit message description within the PR description. For a multi-commit PR I use https://github.com/okurz/scripts/blob/master/git-pr-more to mention each commit message subject line in a list in the PR description. In this special case I added the second commit to the same branch after the PR was initially created from a single PR.

**Pull request merged**